libs/utils/src/hooks/useFetchOmdb.tsx

```diff
@@ -0,0 +1,30 @@
+import { Film } from '@form-exercise/data/interface';
+import axios, { AxiosError } from 'axios';
+import { useQuery } from 'react-query';
+
+interface UseFetchOmdb {
+  searchBy: string;
+  type?: string;
+}
+
+interface Films {
+  Reponse: string;
+  Search: Film[];
+  totalResults: string;
+}
+
+export function useFetchOmdb({ searchBy, type = 'movie' }: UseFetchOmdb) {
```

Comment:
what do you fetch ? Be precise in your naming, it will be easier for you to read you code later and for outsiders to understand what you mean.
- `useGetMovies` for multiple resources of type movie
- `useGetMovie` for one resource of type movie